src/test/resources/features/DepartureFE/DepartureFinal/End-to-end-XI.feature

```diff
@@ -143,7 +144,6 @@ Feature: End to end journey for Northern Ireland office of departure
     # Documents
     When I click the Add documents link on the 'Declaration summary' page
     And I click radio option Yes on the 'Do you want to add any documents?' page
-    And I click radio option No on the 'Do you want to use this document for all items?' page
```

Comment:
I thought this was page was only being removed in the transition journey as per CTCP-3329